src/front-end/src/components/AdminSignup/AdminSignup.js

```diff
@@ -1,132 +1,152 @@
+    createUserInFirebase(values.emailInput, values.password).then(
+      (createUserAttempt) => {
+        if (createUserAttempt.code === 'auth/email-already-in-use') {
+          window.alert(
+            'The email address that you entered is already associated with an email address in our system. Please login to access your account.',
+          );
+        } else {
+          setInDB('users', createUserAttempt.uid, marshallAdminInfo(values));
+          this.setState({ isSubmitted: true });
+        }
+      },
+    );
+  }
 
-  sendEmail() {}
+  onFinishFailed(values) {
+    console.log('Failed submit:', values);
+  }
 
   render() {
-    return (
-      <div className="AdminSignup">
-        <Form
-          onFinish={this.onFinish}
-          onFinishFailed={this.onFinishFailed}
-          autoComplete="off"
-          labelCol={{ span: 3 }}
-          wrapperCol={{ span: 16 }}
-          layout="horizontal"
-          labelAlign="left"
-        >
-          <h1>Register your Organization on Mentor Match</h1>
-
-          <Form.Item
-            label="Organization"
-            name="organization"
-            tooltip="The organization you're planning to be a Mentor with"
-            rules={[{ required: true, message: 'Please input something' }]}
-          >
-            <Input placeholder="Please enter your Organization" />
-          </Form.Item>
-          <Form.Item
-            label="Name"
-            id="nameInput"
-            name="nameInput"
-            // must have an input:
-            rules={[{ required: true, message: 'Please enter your name' }]}
+    if (this.state.isSubmitted) {
+      return <AdminSignup />;
+    }
+    if (!this.state.isSubmitted) {
+      return (
+        <div className="AdminSignup">
+          <Form
+            onFinish={this.onFinish}
+            onFinishFailed={this.onFinishFailed}
+            autoComplete="off"
+            labelCol={{ span: 3 }}
+            wrapperCol={{ span: 16 }}
+            layout="horizontal"
+            labelAlign="left"
           >
-            <Input
-              onInput={this.handleNameChange}
-              id="nameid"
-              placeholder="First name Last name"
-            />
-          </Form.Item>
-          <Form.Item
-            label="Email"
-            id="emailInput"
-            type="email"
-            name="emailInput"
-            // must have an input:
-            rules={[
-              {
-                required: true,
-                message: 'Please enter your email in a proper format',
-              },
-            ]}
-          >
-            <Input
-              onInput={this.handleEmailChange}
-              id="emailid"
-              placeholder="name@example.com"
+            <h1>Register your Organization on Mentor Match</h1>
+
+            <Form.Item
+              label="Organization"
+              name="organizationInput"
+              tooltip="The organization you're planning to be a Mentor with"
```

Comment:
Minor change: we should probably fix the wording of the tooltip to reflect Admins (i.e 'The name of the organization that you're associated with' or something along those lines)